### PR TITLE
DomainTools Iris: Documentation - Fix semgrep errors

### PR DESCRIPTION
**Pull request by wesleya**
Please ensure your pull request (PR) adheres to the following guidelines:

- Please refer to our contributing documentation for any questions on submitting a pull request, link: [Contribution Guide](https://github.com/splunk-soar-connectors/.github/blob/main/.github/CONTRIBUTING.md)

#### Please check if your PR fulfills the following requirements:
- [x] Testing of all the changes has been performed (for bug fixes / features)
- [x] The readme.html has been reviewed and added / updated if needed (for bug fixes / features)
- [x] Use the following format for the PR description: `<App Name>: <PR Type> - <PR Description>`
- [x] Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.
- [x] Verify all checks are passing.
- [x] Do NOT use the `next` branch of the forked repo. Create separate feature branch for raising the PR.
- [x] Do NOT submit updates to dependencies unless it fixes an issue.
#### Please check the type of change your PR introduces:
- [ ] New App
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Documentation
- [ ] Other (please describe): 

## Release Notes (REQUIRED)
- Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.

## What is the current behavior? (OPTIONAL)
- Describe the current behavior that you are modifying.

## What is the new behavior? (OPTIONAL)
- Fixing semgrep errors to pass automated checks. See https://github.com/splunk-soar-connectors/domaintoolsiris/pull/2 for the failed tests.

## Other information (OPTIONAL)
- Any other information that is important to this PR such as screenshots of how the component looks before and after the change.

## Pay close attention to (OPTIONAL)
- Any specific code change or test case points which must be addressed/reviewed at the time of GA release.

## Screenshots (if relevant)

---
Thanks for contributing!

### CODE REVIEW

File: domaintools_iris_connector.py [hunk 1]
```diff
@@ -478,7 +478,7 @@ def _pivot_action(self, param):
             login_url = DomainToolsConnector._get_phantom_base_url() + '/login'
 
             print("Accessing the Login page")
-            r = requests.get(login_url, verify=False)
+            r = requests.get(login_url, timeout=5)
```
Comment (wesleya) [inline domaintools_iris_connector.py:481]: I'm not sure if we had a specific reason for disabling verification? I'm not able to merge to main keeping it set to `False`, but it seems to work fine in my testing without it. Let me know if I'm missing some context here.

`Certificate verification has been explicitly disabled. This permits insecure connections to insecure servers. Re-enable certification validation.`

File: domaintools_iris_connector.py [hunk 2]
```diff
@@ -491,11 +491,11 @@ def _pivot_action(self, param):
             headers['Referer'] = login_url
 
             print("Logging into Platform to get the session id")
-            r2 = requests.post(login_url, verify=False, data=data, headers=headers)
+            r2 = requests.post(login_url, data=data, headers=headers, timeout=5)
             session_id = r2.cookies['sessionid']
         except Exception as e:
             print("Unable to get session id from the platform. Error: " + str(e))
-            exit(1)
+            sys.exit(1)
```
Comment (wesleya) [inline domaintools_iris_connector.py:498]: `Use `sys.exit` over the python shell `exit` built-in. `exit` is a helper for the interactive shell and may not be available on all Python implementations. https://stackoverflow.com/questions/6501121/difference-between-exit-and-sys-exit-in-python`

File: domaintools_iris_connector.py [hunk 3]
```diff
@@ -196,7 +196,7 @@ def _do_query(self, endpoint, action_result, data=None):
             try:
                 # We do not need to display the URL on the phantom UI. Hence, adding it as a debug statement.
                 self.debug_print("GET: {}".format(url))
-                r = requests.get(url)
+                r = requests.get(url, timeout=5)
```
[the remark this answers is not in the document]
Comment (ChuckWoodraska) [reply domaintools_iris_connector.py:199]: Would probably recommend 60 seconds. I think that's a pretty standard default for web servers.

**Review by ChuckWoodraska — APPROVED** [review]

**Comment by ChuckWoodraska** [comment]
This is just a note that I wouldn't worry too much about those changes because in our dev branch we replace the requests with our DT API Wrapper. So eventually these parts will probably go away.

**Pull request merged**